src/advanced/timestamping-tutorial.md

```diff
@@ -0,0 +1,929 @@
+    depends on the Configuration and Time services which should be running when
+    it starts.
+
+The backend part of our Timestamping service is now ready for use.
+However, in the demo on [Github](timestamping) we have taken it a step further
+and addedd frontend.
+
+## Start the Blockchain network
+
+Now that our service is ready, we can start up a netork of four nodes running
+Exonum with the Timestamping service we have created. All these steps are to be
+performed in the directory containing `Cargo.toml`.
+
+1. Install Rust dependencies:
+   ```
+   cargo install
+   ```
+2. Generate blockchain configuration for the network we are setting up:
+   ```
+   mkdir example
+   exonum-timestamping generate-template example/common.toml --validators-count 4
+   ```
+3. Generate templates of node configurations indicating the addresses which the
+   nodes will use for communication.
+   ```
+   exonum-timestamping generate-config example/common.toml  example/pub_1.toml example/sec_1.toml --peer-address 127.0.0.1:6331
+   exonum-timestamping generate-config example/common.toml  example/pub_2.toml example/sec_2.toml --peer-address 127.0.0.1:6332
+   exonum-timestamping generate-config example/common.toml  example/pub_3.toml example/sec_3.toml --peer-address 127.0.0.1:6333
+   exonum-timestamping generate-config example/common.toml  example/pub_4.toml example/sec_4.toml --peer-address 127.0.0.1:6334
+   ```
+4. Finalize the generation of node configurations indicating the following
+   information: the ports which will be used for private and public API; the
+   private configuration file for this node; the file with the node
+   configuration; and the public configuration files of all the validator nodes.
+   If you do not include the public configuration file of a certain node,
+   such a node will be regarded as an auditor by the validator nodes.
+
+   ```
+   exonum-timestamping finalize --public-api-address 0.0.0.0:8200 --private-api-address 0.0.0.0:8091 example/sec_1.toml example/node_1_cfg.toml --public-configs example/pub_1.toml example/pub_2.toml example/pub_3.toml example/pub_4.toml
+   exonum-timestamping finalize --public-api-address 0.0.0.0:8201 --private-api-address 0.0.0.0:8092 example/sec_2.toml example/node_2_cfg.toml --public-configs example/pub_1.toml example/pub_2.toml example/pub_3.toml example/pub_4.toml
+   exonum-timestamping finalize --public-api-address 0.0.0.0:8202 --private-api-address 0.0.0.0:8093 example/sec_3.toml example/node_3_cfg.toml --public-configs example/pub_1.toml example/pub_2.toml example/pub_3.toml example/pub_4.toml
+   exonum-timestamping finalize --public-api-address 0.0.0.0:8203 --private-api-address 0.0.0.0:8094 example/sec_4.toml example/node_4_cfg.toml --public-configs example/pub_1.toml example/pub_2.toml example/pub_3.toml example/pub_4.toml
+   ```
+5. Run the nodes, indicating the file to which the database will be written:
```

Comment:
Need to mention that you need to do these commands in separated terminals

Reply:
Also there is no note about using `RUST_LOG` environment variable to enable logs